tests/test.sh

```diff
@@ -756,8 +756,8 @@ function test-build-and-run-csperf-l3_loc_eq_1()
         set -x
         L3_LOC_ENABLED=1 ./l3_dump.py                                   \
                             --log-file /tmp/l3.c-server-test.dat        \
-                            --binary "./build/${Build_mode}/bin/use-cases/svmsg_file_server"    \
-                            --loc-binary "./build/${Build_mode}/bin/use-cases/client-server-msgs-perf_loc" \
+                            --binary "tests/build/${Build_mode}/bin/use-cases/svmsg_file_server"    \
+                            --loc-binary "tests/build/${Build_mode}/bin/use-cases/client-server-msgs-perf_loc" \
```

Comment:
Same for both ... use: `./tests/build/...`

Reply:
done